src/webchat-ui/components/WebchatUI.tsx

```diff
@@ -1025,6 +1095,85 @@ export class WebchatUI extends React.PureComponent<
 					return `${unseenMessages.length} ${unreadMessages}`;
 			}
 		};
+		const getChatIcon = () => {
```

Comment:
The WebchatUI code is already too large. Please move this function out to a new file.